packages/xdg_directories/lib/xdg_directories.dart

```diff
@@ -63,23 +63,29 @@ List<Directory> _directoryListFromEnvironment(
   }).toList();
 }
 
-Directory _directoryFromEnvironment(String envVar, String fallback) {
-  assert(envVar != null);
-  final String value = _getenv(envVar);
+Directory? _directoryFromEnv(String envVar) {
```

Comment:
Please don't abbreviate `Environment` to `Env` in these two methods. The methods in this class consistently use the former; making them inconsistent with each other hurts readability.